nifi-nar-bundles/nifi-update-attribute-bundle/nifi-update-attribute-processor/src/main/java/org/apache/nifi/processors/attributes/UpdateAttribute.java

```diff
@@ -86,10 +86,9 @@
 public class UpdateAttribute extends AbstractProcessor implements Searchable {
 
 
-    public static final String DO_NOT_STORE_STATE = "do not store state";
-    public static final String STORE_STATE_LOCALLY = "store state locally";
+    public static final String DO_NOT_STORE_STATE = "Do not store state";
```

Comment:
Yes but since this hasn't been released yet I figured it would be safe to modify. Also why I re-opened the ticket instead of creating a new one.